### PR TITLE
tests(rollout): fix integration after wallet deprecation

### PR DESCRIPTION
**Pull request by mroz22**
Some spider webs here. 

There is an integration test for rollout lib whose main purpose was: 
- check if external data (releases.json, firmwares) are in expected place and in expected format
- test incremental update feature 

With wallet deprecation yesterday these tests stopped working as they used wallet.trezor.io as the storage for firmwares and releases.json. This had become outdated anyway long time ago since @trezor/connect-common package took over this. So in fact we are doing integration testing for something that does not match real world usage. 

So, the first commit removes tests that test nothing useful and can not pass now. 
Also I am thinking about removing entire "incremental update" logic from @trezor/rollout. We don't need it right? This was replaced by "intermediary update" feature?

### CODE REVIEW

File: packages/rollout/src/utils/fetch-browser.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import fetch from 'cross-fetch';
 
 export const fetchFirmware = async (url: string) => {
+    console.log('======= ur', url);
```
Comment (matejkriz) [inline packages/rollout/src/utils/fetch-browser.ts]: probably forgotten?

**Review by matejkriz — APPROVED** [review]
Except the console log it looks good to me and makes sense. 

But I'm still not sure what do you mean by  "incremental update" logic. Could you please point me to the code?

**Comment by mroz22** [comment]
> But I'm still not sure what do you mean by "incremental update" logic. Could you please point me to the code?

Basically whole `getBinary` method

https://github.com/trezor/trezor-suite/blob/0d6ed10ed3430ba511afe0798c19ebf264c50af3/packages/rollout/src/index.ts#L209-L250

Imagine that you won't pass `intermediary` arg. In such case, this method would try to fetch firmware defined by `releaseByFirmware` which might be lets say `1.6.3` firmware.
 
https://github.com/trezor/trezor-suite/blob/0d6ed10ed3430ba511afe0798c19ebf264c50af3/packages/rollout/src/index.ts#L243-L245

The problem is, that this firmware is not avaialble in @trezor/connect-common.

**Comment by matejkriz** [comment]
Thanks for the explanation. I see the issue, would be nice to discuss some general rollout refactor properly, but I see it as a relatively lower priority compared to other opened issues.

**Pull request merged**